src/cpuid/src/template/intel/mod.rs

```diff
@@ -40,6 +40,11 @@ pub(crate) fn msrs_to_save_by_cpuid(cpuid: &CpuId) -> HashSet<u32> {
                 msrs.extend(&[$($msr),*]);
             }
         };
+        ($leaf:expr, $index:expr, $reg:tt, $feature_bit:expr, $msr:expr) => {
```

Comment:
No need to keep both patterns here. We can just keep the last one with `$msrs:expr` that would match for both ranges and slices or other iterable expressions in this case.

Reply:
done